parse-jtl.py

```diff
@@ -0,0 +1,134 @@
+#! /usr/bin/env python
```

Comment:
Nitpick: space between `#!` and `/usr/bin/env`. It doesn't really hurt anything.